### PR TITLE
added gdrive_acknowledge_abuse modifier

### PR DESCRIPTION
**Pull request by aliahari**
added setting gdrive_acknowledge_abuse to true to address the issue #4184

> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

**Review by dberenbaum — APPROVED** [review]
Looks great, thank you!

File: content/docs/user-guide/how-to/setup-google-drive-remote.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ $ dvc add data
 ...
 $ dvc remote add --default myremote \
                            gdrive://0AIac4JZqHhKmUk9PDA/dvcstore
+$ dvc remote modify myremote gdrive_acknowledge_abuse true
```
Comment (dberenbaum) [inline content/docs/user-guide/how-to/setup-google-drive-remote.md:37]:
```suggestion
$ dvc remote modify myremote gdrive_acknowledge_abuse true
```

File: content/docs/user-guide/how-to/setup-google-drive-remote.md [hunk 2]
```diff
@@ -233,11 +234,27 @@ credentials files described above, and usually you get it going through the same
...
+the value of the [`gdrive_acknowledge_abuse` option] and set it to `true` using
+the below command:
+
+```cli
+$ dvc remote modify myremote gdrive_acknowledge_abuse true
+```
+
+</admon>
+
+
```
Comment (dberenbaum) [inline content/docs/user-guide/how-to/setup-google-drive-remote.md:250]:
```suggestion

```

**Comment by aliahari** [comment]
thanks!

File: content/docs/user-guide/how-to/setup-google-drive-remote.md [hunk 3]
```diff
@@ -34,6 +34,7 @@ $ dvc add data
 ...
 $ dvc remote add --default myremote \
                            gdrive://0AIac4JZqHhKmUk9PDA/dvcstore
+$ dvc remote modify myremote gdrive_acknowledge_abuse true
```
Comment (jorgeorpinel) [inline content/docs/user-guide/how-to/setup-google-drive-remote.md:37]: OK so we include it here

File: content/docs/user-guide/how-to/setup-google-drive-remote.md [hunk 4]
```diff
@@ -233,11 +234,25 @@ credentials files described above, and usually you get it going through the same
 authentication process. If `GDRIVE_CREDENTIALS_DATA` is set, the
 `gdrive_user_credentials_file` value (if provided) is ignored.
 
+<admon type="info">
+
+If you recieved the error _This file has been identified as malware or spam and
+cannot be downloaded_ running a `dvc push` or `dvc pull` command you can modify
+the value of the [`gdrive_acknowledge_abuse` option] and set it to `true` using
```
Comment (jorgeorpinel) [inline content/docs/user-guide/how-to/setup-google-drive-remote.md:241]: So why is this note needed?

**Review by jorgeorpinel — COMMENTED** [review]
Just a question... 👇🏼

**Comment by jorgeorpinel** [comment]
p.s. followed up in #4189 (cc @dberenbaum)

**Pull request merged**